### PR TITLE
document simple production update process

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

File: openshift/README.md [hunk 1]
```diff
@@ -73,6 +73,28 @@ The best way to publish an image is to make a new release of the project,
...
+quay repository](https://quay.io/repository/log-detective/website?tab=history),
+you can update the website.
+
+The easiest way is to delete the existing running pod (yes, scary 😱) and let
+OpenShift fetch the newest image and use it to start a new pod.
+
+```
+oc get pods
+oc delete pod log-detective-website-xxxxxxxx-xxxxx
+```
```
Comment (FrostyX) [inline openshift/README.md:88]: Would `oc rollout restart deploy/log-detective-website` work instead of deleting?

Comment (TomasTomecek) [reply openshift/README.md:88]: from the `--help` page:
```
 There are several deployment strategies defined:

  *  Rolling (default) - scales up the new replication controller in stages, gradually reducing the number of old pods.
If one of the new deployed pods never becomes "ready", the new rollout will be rolled back (scaled down to zero). Use
when your application can tolerate two versions of code running at the same time (many web applications, scalable
databases)
  *  Recreate - scales the old replication controller down to zero, then scales the new replication controller up to
full. Use when your application cannot tolerate two versions of code running at the same time
```

yes, that should work and we already have that documented on the lin 130

https://github.com/fedora-copr/log-detective-website/pull/111/files#diff-10a0fba9e0f5927439e2c632e20443232a4578c9be2df85bd25f6f768ebf5c1eR130

**Comment by TomasTomecek** [comment]
I guess we can close this then 😅

**Comment by TomasTomecek** [comment]
@FrostyX @FrostyX rereview please

**Review by FrostyX — APPROVED** [review]

**Pull request merged**